develop/components/pages/CarouselPage.tsx

```diff
@@ -19,39 +16,77 @@ const CarouselPage: React.FunctionComponent = () => {
 
                 <div className="info">
                     <h2>Output</h2>
-                    <p>Here are sample outputs</p>
+                    <p>
+                        Carousel with <b>slide</b> transition
+                    </p>
                     <div className="result wide">
-                        <Carousel
-                            list={carouselList}
-                            afterChange={() => {
-                                console.log("changed");
-                            }}
-                            autoPlay={true}
-                            infinite={true}
-                        />
+                        <CarouselSample />
+                    </div>
+
+                    <p>
+                        Carousel with <b>fade</b> transition
+                    </p>
+                    <div className="result wide">
+                        <CarouselSample transitionStyle="fade" />
                     </div>
                 </div>
             </div>
         </div>
     );
 };
 
-const carouselList: Array<CarouselItem> = [
-    {
-        title: "Ipsum consequat nisl",
-        desc: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
-        image: firstImg,
-    },
-    {
-        title: "Interdum velit euismod",
-        desc: "Lectus quam id leo in",
-        image: secondImg,
-    },
-    {
-        title: "Risus in hendrerit",
-        desc: "Augue eget arcu dictum varius",
-        image: thirdImg,
-    },
-];
+const CarouselSample: React.FC<CarouselProps> = React.memo((props: CarouselProps) => {
+    return (
+        <Carousel
+            {...props}
+            showIndicators
+            list={[
+                {
+                    children: (
+                        <div className="p-5 bg-primary">
+                            <h1 className="text-light text-center">Start</h1>
+                            <p className="text-light text-center">
+                                <a className="text-light" href="#">
+                                    test
+                                </a>
+                            </p>
+                        </div>
```

Comment:
I know this is example, but maybe you can write this block in a simple function component to reuse it in line `59-66`, `69-77`, `79-86`

Reply:
done